source/_posts/2016-07-29-ember-qunit-nice-errors.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "ember-qunit-nice-errors"
+category: ember
+comments: true
+author:
+  name: Samanta de Barros
+  email: samanta@wyeworks.com
+  twitter_handle: sami_dbc
+  github_handle: sdebarros
+  image: /images/team/samanta-de-barros.jpg
+  description: Software Engineer
+---
+
+As any Ember developer probably knows by now, the error messages shown by failing asserts are far from ideal. Getting a **failed, expected argument to be truthy, was: false** followed by a stack trace that points to lines on the generated test file (one file with **all** your compiled tests), can be pretty disappointing when you want to be able to get quick feedback from your tests.
```

Comment:
Hmm.. it's part of the output, not the actual output and I'm using it to highlight something "bad" more than showing a result, I think is more relatable as text, but I'll see how it looks as inline code